src/components/profile/profile.style.js

```diff
@@ -0,0 +1,44 @@
+import styled, { css } from 'styled-components';
+import classicProfileStyle from './profile-classic.style';
+import Portrait from '../portrait/portrait';
+import baseTheme from '../../style/themes/base';
+
+const ProfileNameStyle = styled.span`
+    font-weight: bold;
+`;
+
+const ProfileStyle = styled.div`
+    white-space: nowrap;
+    color: ${({ theme }) => theme.text.color};
+
+    ${({ large }) => large && css`
+        ${ProfileNameStyle} {
+            font-size: 20px;
+            font-weight: 400;
+            line-height: 21px;
+        }
+    `}
+    ${classicProfileStyle}
```

Comment:
could you add a blank line to separate `classic` from the rest?

Reply:
There is no more classic import